src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,118 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    public static final int DEFAULT_CAPACITY = 10;
+    public static final int ARRAY_GROWTH_VALUE = 1;
```

Comment:
The `ARRAY_GROWTH_VALUE` constant is set to 1, but the task requires the array to grow by 1.5 times when it is full. Consider changing the logic in `growIfArrayIsFull` to multiply the current capacity by 1.5 instead of using a bitwise shift.